main.go

```diff
@@ -772,6 +772,49 @@ func (t *TPUWebhookServer) mutatePod(admissionReview *admissionv1.AdmissionRevie
 				}
 				patches = append(patches, namePatch)
 			}
+
+			// inject NODE_ADDR. This Env Var is required for TPU_DEVICE_PLUGIN_ADDR
+			if getEnvironmentVariable("NODE_ADDR", container) == "" {
+				klog.V(1).InfoS("mutatePod, add NODE_ADDR", "RayCluster", namespace+"/"+clusterName)
+				tpuDevicePluginAddr := corev1.EnvVar{
+					Name: "NODE_ADDR",
```

Comment:
Let's keep this env var specifci to TPU device plugin to avoid accidental overwrite. Maybe something like `TPU_DEVICE_PLUGIN_HOST_IP`